### PR TITLE
Add vorbis comment picture support

### PR DESCRIPTION
**Pull request by wader**

### CODE REVIEW

**Comment by wader** [comment]
Fixes #64

File: vorbis.go [hunk 1]
```diff
@@ -56,6 +58,15 @@ func (m *metadataVorbis) readVorbisComment(r io.Reader) error {
 		}
 		m.c[strings.ToLower(k)] = v
 	}
+
+	if b64data, ok := m.c["metadata_block_picture"]; ok {
+		data, err := base64.StdEncoding.DecodeString(b64data)
+		if err != nil {
+			return err
+		}
+		m.readPictureBlock(bytes.NewBuffer(data))
```
Comment (dhowden) [inline vorbis.go:67]:
```suggestion
		m.readPictureBlock(bytes.NewReader(data))
```

Comment (wader) [reply vorbis.go]: Ah doh

File: cmd/tag/main.go [hunk 2]
```diff
@@ -96,4 +97,6 @@ func printMetadata(m tag.Metadata) {
 	fmt.Printf(" Picture: %v\n", m.Picture())
 	fmt.Printf(" Lyrics: %v\n", m.Lyrics())
 	fmt.Printf(" Comment: %v\n", m.Comment())
+
+	ioutil.WriteFile("bla", m.Picture().Data, 0777)
```
Comment (dhowden) [inline cmd/tag/main.go:101]: This make it in by mistake?

Comment (wader) [reply cmd/tag/main.go]: Yeap! reverted, too fast with the git add -u

**Pull request merged**